src/AspNet.Security.OpenId/OpenIdAuthenticationOptions.cs

```diff
@@ -104,7 +106,7 @@ public new OpenIdAuthenticationEvents Events
         /// When this property is set to <c>null</c>, the data protection provider
         /// is directly retrieved from the dependency injection container.
         /// </summary>
-        public IDataProtectionProvider DataProtectionProvider { get; set; }
+        public new IDataProtectionProvider DataProtectionProvider { get; set; }
```

Comment:
Remove this property.

Reply:
why should I remove this property? the handler requires state parameter.

Reply:
Because this property is now defined in the base class in 2.0 (which is why you need the `new` keyword to hide it).